app/models/claim_va_notification.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ClaimVANotification < ApplicationRecord
+  belongs_to :saved_claim
```

Comment:
does this need a relationship on SavedClaim also? can/should it have one?

Reply:
@wayne-weibel I thought the idea was to have the saved_claim_id be the FK to claim. I guess it doesn't need to be directly involved with saved_claim. I can remove it